backend/corpora/lambdas/api/v1/curation/collection_uuid/dataset.py

```diff
@@ -0,0 +1,19 @@
+from flask import g
+
+from backend.corpora.api_server.db import dbconnect
+from backend.corpora.common.entities import Dataset
+from backend.corpora.common.utils.http_exceptions import InvalidParametersHTTPException
+from backend.corpora.lambdas.api.v1.dataset import delete_dataset_common
+
+
+@dbconnect
+def delete_dataset(token_info: dict, collection_uuid: str, curator_tag: str = None, dataset_uuid=None):
+    db_session = g.db_session
+    if dataset_uuid:
+        dataset = Dataset.get(db_session, dataset_uuid, include_tombstones=True)
```

Comment:
For my own edification... why do we want to include tombstoned datasets here again? 🤔

Reply:
if they try to delete a dataset that has already been deleted we will return a Sucess rather than an error.

Reply:
is that the standard way to do that? I'm unfamiliar.